bazaar/interface.rst

```diff
@@ -1,12 +1,207 @@
 
-Bazaar Interfaces
-*****************
+Edustore Distributed API
```

Comment:
We should use the term Bazaar here as it is defined elsewhere to describe the service which handles material transfer, commerce and licensing. Also "Edustore" is product name of one company.